python/dgl/subgraph.py

```diff
@@ -40,102 +37,8 @@ class DGLSubGraph(DGLGraph):
     """
     def __init__(self, parent, sgi, shared=False):
         super(DGLSubGraph, self).__init__(graph_data=sgi.graph,
-                                          readonly=True)
+                                          readonly=True,
+                                          parent=parent,
+                                          sgi=sgi)
```

Comment:
Could you help remove `shared`? Please also modify the docstring accordingly. And why do we still have this class?

Reply:
removed